.travis.yml

```diff
@@ -0,0 +1,31 @@
+sudo: false    #container 
```

Comment:
Maybe add a comment block to the top of the file describing what it is and a link to the travis configuration guide online?